### PR TITLE
 Smart mainline PR from niroj's smart-mainline branch solving merge conflict

### PR DESCRIPTION
**Pull request by nirojshrestha019**
Issue #, if available:

Description of changes:

Checklist:

* [x] Have you successfully deployed to an AWS account with your changes?
* [x] Have you written new tests for your core changes, as applicable?

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by lquanx** [comment]
looks good to me.

File: src/authZConfig.ts [hunk 1]
```diff
@@ -22,7 +22,7 @@ export const scopeRule: ScopeRule = {
 };
```
Comment (lwarren01) [inline src/authZConfig.ts:22]: We might want to make the same change to the system.write array that was made for user.write, the addition of 'delete' to the list.

**Review by jmrah — APPROVED** [review]

**Review by lquanx — APPROVED** [review]

**Comment by lquanx** [comment]
Run 'yarn lint-fix' to fix any code style  issues. Otherwise codepipeline deployment would fail, if there are any lint issues

**Comment by nirojshrestha019** [comment]
> 
> 
> Run 'yarn lint-fix' to fix any code style issues. Otherwise codepipeline deployment would fail, if there are any lint issues

done.

**Pull request merged**